### PR TITLE
feat: adding controller.

### PR DESCRIPTION
**Pull request by egp1020**

### CODE REVIEW

File: src/main/java/culturemedia/controller/CultureMediaController.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
+package culturoteca.controllers;
+
+import java.util.*;
+
+import culturoteca.exception.VideoNotFoundException;
```
Comment (Xiomara18b) [inline src/main/java/culturemedia/controller/CultureMediaController.java:5]: is : import culturemedia.exception.VideoNotFoundException;

File: src/main/java/culturemedia/controller/CultureMediaController.java [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+import culturoteca.model.Video;
```
Comment (Xiomara18b) [inline src/main/java/culturemedia/controller/CultureMediaController.java:6]: is : import culturemedia.model.Video;

File: src/main/java/culturemedia/controller/CultureMediaController.java [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+import culturoteca.service.impl.CultureMediaServiceImpl;
```
Comment (Xiomara18b) [inline src/main/java/culturemedia/controller/CultureMediaController.java:7]: is : import culturemedia.service.impl.CultureMediaServiceImpl;

File: src/main/java/culturemedia/controller/CultureMediaController.java [hunk 4]
```diff
@@ -0,0 +1,26 @@
+package culturoteca.controllers;
```
Comment (Xiomara18b) [inline src/main/java/culturemedia/controller/CultureMediaController.java]: the package name is controller, no controllers and culturemedia, not culturoteca

File: src/main/java/culturemedia/controller/CultureMediaController.java [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+    private final CultureMediaServiceImpl cultureMediaService;
+
+
+    public CultureMediaController(CultureMediaServiceImpl cultureMediaService) {
+        this.cultureMediaService = cultureMediaService;
+    }
+
+
+    public List<Video> find_allVideos() throws VideoNotFoundException {
+        List<Video> videos = null;
```
Comment (Xiomara18b) [inline src/main/java/culturemedia/controller/CultureMediaController.java]: initialization videos = null; unnecesary

File: src/main/java/culturemedia/controller/CultureMediaController.java [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+        videos = cultureMediaService.findAll();
```
Comment (Xiomara18b) [inline src/main/java/culturemedia/controller/CultureMediaController.java]: the correct form is to specify that videos is a List of video = cultureMediaService.findAll()

File: src/main/java/culturemedia/controller/CultureMediaController.java [hunk 7]
```diff
@@ -0,0 +1,26 @@
...
+
+    private final CultureMediaServiceImpl cultureMediaService;
+
+
+    public CultureMediaController(CultureMediaServiceImpl cultureMediaService) {
+        this.cultureMediaService = cultureMediaService;
+    }
+
+
+    public List<Video> find_allVideos() throws VideoNotFoundException {
```
Comment (Xiomara18b) [inline src/main/java/culturemedia/controller/CultureMediaController.java]: the name should be : findAllVideos()

File: src/main/java/culturemedia/controller/CultureMediaController.java (hunk 3 above)
Comment (Xiomara18b) [inline src/main/java/culturemedia/controller/CultureMediaController.java]: is : import culturemedia.service.CultureMediaService;
the CultureMediaService interface is imported, not the implementation

File: src/main/java/culturemedia/controller/CultureMediaController.java [hunk 8]
```diff
@@ -0,0 +1,26 @@
...
+
+import java.util.*;
+
+import culturoteca.exception.VideoNotFoundException;
+import culturoteca.model.Video;
+import culturoteca.service.impl.CultureMediaServiceImpl;
+
+public class CultureMediaController {
+
+    private final CultureMediaServiceImpl cultureMediaService;
```
Comment (Xiomara18b) [inline src/main/java/culturemedia/controller/CultureMediaController.java]: the interface is used, not the impl

File: src/main/java/culturemedia/controller/CultureMediaController.java (hunk 2 above)
Comment (Xiomara18b) [inline src/main/java/culturemedia/controller/CultureMediaController.java]: is : import culturemedia.model.Video;

File: src/main/java/culturemedia/controller/CultureMediaController.java (hunk 1 above)
Comment (Xiomara18b) [inline src/main/java/culturemedia/controller/CultureMediaController.java]: is : import culturemedia.exception.VideoNotFoundException;

File: src/main/java/culturemedia/controller/CultureMediaController.java [hunk 9]
```diff
@@ -0,0 +1,26 @@
...
+        this.cultureMediaService = cultureMediaService;
+    }
+
+
+    public List<Video> find_allVideos() throws VideoNotFoundException {
+        List<Video> videos = null;
+        videos = cultureMediaService.findAll();
+        return videos;
+    }
+
```
Comment (Xiomara18b) [inline src/main/java/culturemedia/controller/CultureMediaController.java:20]: missing method to add videos

**Review by Xiomara18b — CHANGES_REQUESTED** [review]
Please review and change

**Comment by egp1020** [comment]
Cambios realizados, muchas gracias por tu aporte.

**Pull request merged**